client/src/domain/MovieDetails/index.js

```diff
@@ -0,0 +1,31 @@
+import './style.scss';
+import React from 'react';
+import poster from '../../noTimeToDiePoster.jpg';
+
+function MovieDetails({
+  movie = {
+    title: 'No Time to Die',
+    duration: 163,
+    releaseDate: '30 September 2021 (UK)',
+    description:
+      'James Bond has left active service. His peace is short-lived when Felix Leiter, an old friend from the CIA, turns up asking for help, leading Bond onto the trail of a mysterious villain armed with dangerous new technology.',
+    genre: 'Action, Adventure, Thriller',
+  },
+}) {
+  return (
+    <div className="movie-details">
```

Comment:
Perhaps you can implement more BEM-like classes, ex. movie__details or movie__img